src/promptflow-tools/promptflow/tools/common.py

```diff
@@ -18,6 +19,7 @@
 
 
 GPT4V_VERSION = "vision-preview"
+VALID_ROLES = ["assistant", "function", "user", "system"]
```

Comment:
do we need to add "tool" role here?

Reply:
We add tool support in PR https://github.com/microsoft/promptflow/pull/2223/files

Prefer not to do in this PR, because tool role support is complicated, not just simply adding a role here.